### ISSUE TITLE
PCAs of methylation and gene exp data

### ISSUE THREAD

**Issue by yaaminiv** [sr320/ceabigr#73]
@kubu4 @sr320 Where can I find the PCAs of methylation and gene exp data? Looking for sex-specific PCAs ideally, but would take PCAs of all data

**Comment by kubu4** [comment]
Like in this Mixomics "testing" I ran through?

https://rpubs.com/kubu4/cvir-gonad-oa-mixomics_testing

**Comment by yaaminiv** [comment]
@kubu4 yes! is there any way you could color the PCAs by sex and treatment? those would work great

**Comment by kubu4** [comment]
> is there any way you could color the PCAs by sex and treatment

Can you clarify exactly what you have in mind?

On those plots, they're colored by sex and then shapes designate treatment. 

Are you wanting four colors? Do you want the shapes to remain?

And, after all those questions, I honestly have no idea if I can figure that out! LOL! I just followed the Mixomics tutorial, so would have to dig much deeper to see if I can figure out other ways to manipulate the plots....

**Comment by kubu4** [comment]
Also, just to note, gene expression _and_ methylation are only taken into account for the sparse Projection to Latent Structure (sPLS), _not_ in the PCAs.

sPLS tutorial I followed is here:

https://mixomicsteam.github.io/Bookdown/pls.html

**Comment by yaaminiv** [comment]
> On those plots, they're colored by sex and then shapes designate treatment.

ah never mind! i didn't see the different shapes
